socialhome/streams/app/components/Stream.vue

```diff
@@ -1,30 +1,27 @@
 <template>
     <div>
         <div class="container-flex">
-            <div v-show="$store.state.stream.hasNewContent" class="new-content-container">
+            <div v-show="$store.state.hasNewContent" class="new-content-container">
```

Comment:
Change this class to an `id` and you don't have to duplicate the CSS. Currently in the non-Vue version this is an `id`.

Reply:
There are two main reasons for using classes here:

1. classes are more efficient than ids for applying CSS (see for instance answer [here](https://stackoverflow.com/questions/7623947/css-id-class-vs-class-performance-which-is-better#7623970)),
2. many of the ids you created are bound to existant JS code, which interfere with Vue rewriting.

Reply:
Ah right, the existing JS 👍